Cargo.toml

```diff
@@ -7,3 +7,5 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+clap = { git = "https://github.com/clap-rs/clap/" }
```

Comment:
They have a prerelease on crates.io!
`clap = "3.0.0-beta.2"`

Reply:
Good catch! Fixed in 19cb46f.